### ISSUE TITLE
Implement proper cleanup procedures to avoid memory leaks during undeployment in WildFly

### ISSUE THREAD

**Issue by jmartisk** [smallrye/smallrye-graphql#405]

**Comment by t1** [comment]
@jmartisk: is this issues solved and can be closed?

**Comment by jmartisk** [comment]
@t1 no, I still haven't had much time to look into it and there definitely are some leaks. Because support for multiple MP applications in WildFly is generally best-effort only, this is not very high priority.
